airflow/api_connexion/endpoints/dag_endpoint.py

```diff
@@ -77,7 +77,7 @@ def get_dags(limit, session, offset=0, only_active=True, tags=None, dag_id_patte
         cond = [DagModel.tags.any(DagTag.name == tag) for tag in tags]
         dags_query = dags_query.filter(or_(*cond))
 
-    total_entries = len(dags_query.all())
+    total_entries = dags_query.count()
```

Comment:
I don’t think this is an issue here because `dags_query` does not have things like JOIN and shouldn’t need to be deduplicated. And even if it is, we arguably _should_ add deduplication logic to `dags_query` (otherwise offset and limit won’t work right), which will make `count()` correct in the end.